next_generation/requirements.txt

```diff
@@ -8,7 +8,8 @@ itsdangerous==1.1.0
 Jinja2==2.11.3
 jmespath==0.10.0
 MarkupSafe==1.1.1
-psycopg2==2.8.6
+#psycopg2==2.8.6
+psycopg2-binary
```

Comment:
we don't need psycopg2-binary as .staging/ already contains the pre-built binaries.